master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -172,6 +174,14 @@ public void updateWorkerHeartbeatMeta(
             info.lastHeartbeat_$eq(time);
           });
     }
+    partitionWritten.set(
```

Comment:
You just use the resource consumption from 1 worker and there might be workers. 

The correct way to get the values is to sum diskBytesWritten and diskFileCount from all the latest workerHeartBeat snapshots.